source/app/powertabeditor.cpp

```diff
@@ -1366,6 +1367,28 @@ void PowerTabEditor::editRepeatEnding()
     }
 }
 
+void PowerTabEditor::updateDynamic(Dynamic::VolumeLevel volume)
+{
+    ScoreLocation &location = getLocation();
+    const Dynamic *dynamic = ScoreUtils::findByPosition(
+                location.getStaff().getDynamics(), location.getPositionIndex());
+
+    if (dynamic)
+    {
+        myUndoManager->push(new EditDynamic(location, volume, dynamic),
+                            location.getSystemIndex());
+    }
+    else
+    {
+        if (volume == dynamic->getVolume())
```

Comment:
I think this check should move into the `if ()` block above, since it applies if there is already an existing dynamic at the location.